src/app/src/main/kotlin/org/imperial/mrc/hint/FileManager.kt

```diff
@@ -61,12 +64,19 @@ class LocalFileManager(
 
     override fun getFile(type: FileType): SessionFileWithPath? {
         return sessionRepository.getSessionFile(session.getId(), type)
-                ?.toSessionFileWithPath(appProperties.uploadDirectory)
+                ?.toSessionFileWithPath(uploadPath)
     }
 
-    override fun getAllFiles(): Map<String, String> {
+    override fun getAllHashes(): Map<String, String> {
         val hashes = sessionRepository.getHashesForSession(session.getId())
-        return hashes.mapValues { "${appProperties.uploadDirectory}/${it.value}" }
+        return hashes.mapValues { "$uploadPath/${it.value}" }
+    }
+
+    override fun getAllFiles(vararg include: FileType): Map<String, SessionFileWithPath> {
```

Comment:
Feels like this should probably be called getFiles rather than getAllFiles if it optionally takes a param to define what files to return.